src/tokens/utils.ts

```diff
@@ -87,6 +87,22 @@ export function sortFirstStableThenCeloThenOthersByUsdBalance(
   return usdBalance(token2).comparedTo(usdBalance(token1))
 }
 
+export function sortCicoTokens(token1: TokenBalance, token2: TokenBalance): number {
+  const cicoTokenInfo = getDynamicConfigParams(
+    DynamicConfigs[StatsigDynamicConfigs.CICO_TOKEN_INFO]
+  )
+  if (!cicoTokenInfo.cicoOrder[token1.tokenId] && !cicoTokenInfo.cicoOrder[token2.tokenId]) {
+    return sortFirstStableThenCeloThenOthersByUsdBalance(token1, token2)
+  }
+  if (!cicoTokenInfo.cicoOrder[token1.tokenId]) {
+    return 1
+  }
+  if (!cicoTokenInfo.cicoOrder[token2.tokenId]) {
+    return -1
+  }
+  return cicoTokenInfo.cicoOrder[token1.tokenId] < cicoTokenInfo.cicoOrder[token2.tokenId] ? -1 : 1
```

Comment:
It's possible that multiple tokens map to the same value, right? It seems like the dynamic config just specifies token "priority", rather than actual position in the list, so it's possible that two tokens have the same priority? Can we handle the case where the values are the same here?

Reply:
good catch, I think hopefully the dynamic config will have unique numbers for the different tokens so it can sort correctly, but I added a safeguard in case they are the same.